### PR TITLE
Add routing table for traffic arriving on public network

### PR DESCRIPTION
**Pull request by jovial**
This removes routing asymmetry when traffic bound for the public network

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/controllers/network [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Public network IP routing rules.
+public_rules:
+  # Use the public routing table for all traffic originating from the public
+  # network.
+  - "from {{ public_cidr }} table public"
+
+# Public network static IP routes.
+public_routes:
+  # Default route via the firewall's public network interface, added to the
```
Comment (markgoddard) [inline etc/kayobe/inventory/group_vars/controllers/network:21]: Is there a firewall?

File: etc/kayobe/inventory/group_vars/controllers/network [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  # public routing table.
+  - cidr: "0.0.0.0/0"
+    gateway: "10.12.9.1"
```
Comment (oneswig) [inline etc/kayobe/inventory/group_vars/controllers/network:24]: Is this gateway IP parameterised anywhere?

**Review by markgoddard — APPROVED** [review]

File: etc/kayobe/inventory/group_vars/controllers/network [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# List of IP routing tables. Each item should be a dict containing 'id' and
+# 'name' items. These tables will be added to /etc/iproute2/rt_tables.
+network_route_tables:
+  # IP routing table for traffic orginating from the public network. Because of
```
Comment (oneswig) [inline etc/kayobe/inventory/group_vars/controllers/network]: Is it more accurate to say, routed from our public IP?  In this specific case, the traffic from eod-seed is originating from 10.12.3.2(IIRC) and is routed to 10.12.9.254.  Our response, from 10.12.9.254, needs to be routed via 10.12.9.1

**Review by oneswig — APPROVED** [review]
Good stuff, assume it works...

File: etc/kayobe/inventory/group_vars/controllers/network (hunk 1 above)
Comment (oneswig) [reply etc/kayobe/inventory/group_vars/controllers/network:21]: Not one that we control as part of this deployment but there is I believe a Fortinet device.

File: etc/kayobe/inventory/group_vars/controllers/network (hunk 2 above)
Comment (jovial) [reply etc/kayobe/inventory/group_vars/controllers/network:24]: Not that I am aware of.

**Pull request merged**